app/models/family_member.rb

```diff
@@ -168,9 +168,14 @@ def self.find(family_member_id)
 
   private
 
+  def publish_private_family_member_created_event
+    ::Operations::Private::FamilyMemberCreated.new.call(self)
```

Comment:
@jacobkagon This will add load on the callback. We want to avoid these kinds of situations and for the same reason you had to refactor the person save callback. I suggest you publish a family member created private event with the required information and construct the CvFamily asynchronously.

I am expecting a similar implementation that you did for the person save callback.